ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh

```
@@ -348,6 +347,12 @@ if [[ -n "${GUEST_FEATURE_SET}" ]]; then
   fi
 fi
 
+# Create the kubeconfig symlink only AFTER all readiness checks (clusteroperators,
+# NodePools, feature set) have passed.  The EphemeralCluster controller watches for
+# this file to set the ClusterReady condition; placing it earlier causes tests to
+# start before the cluster is actually ready, leading to DNS resolution failures.
+ln -s ${SHARED_DIR}/nested_kubeconfig ${SHARED_DIR}/kubeconfig
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)